src/lib/constants.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+export default {
+    'sevenDays': 6.048e+8,
```

Comment:
Personally, (nitpick) I think its easier to understand if you wrote it like this: `sevenDays: 1000*60*60*24*7`

😆